packages/components/src/internal/form.ts

```diff
@@ -201,9 +202,24 @@ export class FormControlController implements ReactiveController {
       });
     }
 
+    // Validate the form and prevent submission if it's invalid
     if (this.form && !this.form.noValidate && !disabled && !reportValidity(this.host)) {
       event.preventDefault();
       event.stopImmediatePropagation();
+
+      const invalidElements: NodeListOf<HTMLFormElement> | undefined = this.form?.querySelectorAll('[data-invalid]');
+
+      // Check for any radio groups, we need to dispatch the invalid event on them manually
```

Comment:
Check out the Solid Form sample for context.  Without this alteration, the inline error message will not show up on an invalid submission.